engines/draci/animation.cpp

```diff
@@ -436,7 +436,7 @@ void AnimationManager::deleteAnimation(Animation *anim) {
 		}
 	}
 
-	// Iterate the second time to decrease indexes greater than the deleted animation index
```

Comment:
I don't think this should be changed. Both forms are valid and (even in the scummvm codebase) "indexes" is used in thousands of places

Reply:
Granted, but since i plan to go over the complete codebase, consistency will be reached again eventually.

Also, both are valid, yes, but indexes relate to non-technical use, as in indexes in books for example, while indices is meant to be a more technical description of things, like in math based operations.
Since nearly all of the index stuff in this program code is math-based, indices seems to be more fitting.

but that, of coure, is pure subjective

Reply:
I know that "indices" is preferable, especially in mathematics. But in source code "indexes" is much more frequent, and since changing all the variables names is out of the question, it looks a bit inconsistent to use "indices" in the comments.
But as you said, it's completely subjective

Reply:
@tag2015 @raziel- : Speaking as a native British English speaker (though I know the project uses American English as standard), I would highly recommend not changing this.

While it is true that more technical language uses indice / indices, these are highly obtuse and uncommon compared to index / index(e)s.

It should also be noted that indexes is actually the more common way of pluralising index, since the "xs" grouping looks wrong to most English natives. In any case, it is a perfectly acceptable ortographic variation and possibly the more common one.

See: https://www.nasdaq.com/articles/indexes-or-indices-whats-the-deal-2016-05-12

Reply:
You should also be aware that some games / source code may be British English, not American English and there are more than a few differences in word choice / spelling:
https://en.wikipedia.org/wiki/American_and_British_English_spelling_differences

Reply:
doing a quick search over the code brings up 260+ instances of "indices" (variables and comments) compared to a rough 100 cases of  "indexes".

When i started, there obviously were a little more of the latter and i thought (stupid idea, i know) that consistency would be a basic aim.

seems i was wrong, i think it would be best if i completely skip "indexes" for future PRs

@bluegr 

please advise

Reply:
> doing a quick search over the code brings up 260+ instances of "indices" (variables and comments) compared to a rough 100 cases of "indexes".
> 

Those numbers are way too low. Are you including only comments, or excluding composed words such as sequenceIndexes , spriteIndexes and such?
Searching for "indexes" only in /engines, I get about 10k occurrences vs about 1.5k of "indices"

Reply:
heh, seems this qas onlythe first of many pages :-)